### PR TITLE
Autographamt api code- Added api calls to get whole bible, book and chapter

### PR DESCRIPTION
**Pull request by premkumaar**
Added a api call to fetch 
-the whole bible
https://vachanonline.com/v1/bibles/${bibleID=SourceID}/json OR usfm 
-a whole book
https://vachanonline.com/v1/bibles/${bibleID=SourceID}/books/${biblebookCode}/json OR usfm
-a single chapter
https://vachanonline.com/v1/bibles/${bibleID=SourceID}/books/${biblebookCode}/chapters/{chapterId} 
as per documentation in 
https://bridgeconn-my.sharepoint.com/:w:/p/reena_singh/EXMkZvBGNPdDjitoJIjAR34BorHYB866eRvZBPy2Kp7miQ?rtime=d2_5qmgZ10g

### CODE REVIEW

File: agmt/main.py [hunk 1]
```diff
@@ -45,7 +45,7 @@
 host_api_url = os.environ.get("AGMT_HOST_API_URL")
 host_ui_url = os.environ.get("AGMT_HOST_UI_URL")
 system_email = os.environ.get("MTV2_EMAIL_ID", "autographamt@gmail.com")
-
+print(postgres_user,postgres_database,postgres_host)
```
Comment (bijob89) [inline agmt/main.py]: Please remove this statement

File: agmt/main.py [hunk 2]
```diff
@@ -1880,7 +1880,52 @@ def getBibleBooks(sourceId):
     return json.dumps(bibleBooks)
 
 
+@app.route("/v1/bibles/<sourceId>/<c_type>", methods=["GET"])
```
Comment (bijob89) [inline agmt/main.py]: It would be better to rename `c_type` to `format`

File: agmt/main.py [hunk 3]
```diff
@@ -1880,7 +1880,52 @@ def getBibleBooks(sourceId):
...
+def getBible(sourceId, c_type):
```
Comment (bijob89) [inline agmt/main.py]: Rename `c_type` to `format`

File: agmt/main.py [hunk 4]
```diff
@@ -1880,7 +1880,52 @@ def getBibleBooks(sourceId):
...
+    elif c_type.lower() == 'json':
+        usfmText = {"sourceId":sourceId,"bibleContent":rst[0]["parsedJson"]}
+    else:
+        return '{"success": false, "message":"Invalid Content Type"}'
+    cursor.close()
+    return json.dumps(usfmText)
+    
 
+@app.route("/v1/bibles/<sourceId>/books/<book_code>/<c_type>", methods=["GET"])
+def getBook(sourceId,book_code, c_type):
```
Comment (bijob89) [inline agmt/main.py]: Rename `c_type` to `format` for two lines.

File: agmt/main.py [hunk 5]
```diff
@@ -1926,7 +1971,29 @@ def getBibleChapters(sourceId, biblebookCode):
     except Exception as ex:
         return '{"success": false, "message":"%s"}' %(str(ex))
 
-
+@app.route("/v1/bibles/<sourceId>/books/<book_code>/chapter/<chapter_id>", methods=["GET"])
+def getChapter(sourceId,book_code,chapter_id):
```
Comment (bijob89) [inline agmt/main.py]: As a convention, please use Camel Case for all variables.

File: agmt/main.py (hunk 2 above)
Comment (premkumaar) [reply agmt/main.py]: renamed c_type to format

File: agmt/main.py (hunk 3 above)
Comment (premkumaar) [reply agmt/main.py]: renamed c_type to format

File: agmt/main.py (hunk 4 above)
Comment (premkumaar) [reply agmt/main.py]: renamed c_type to format

File: agmt/main.py (hunk 5 above)
Comment (premkumaar) [reply agmt/main.py]: changed all variables to Camel Case

**Comment by premkumaar** [comment]
all Pull Request changes done, please review.

**Pull request merged**